### PR TITLE
[access-15] Change landing page to dashboard

### PR DESCRIPTION
**Pull request by litocabornay**
These changes able to change landing page to dashboard. Please see sub tasks.

### CODE REVIEW

**Comment by cjestal** [comment]
lits, pakitanggal sa mga commented codes ug white lines

**Comment by litocabornay** [comment]
removed commented codes and white lines.

File: resources/views/home.blade.php [hunk 1]
```diff
@@ -3,15 +3,134 @@
...
-        <div class="col-md-8 col-md-offset-2">
+        <!-- <div class="col-md-8 col-md-offset-2">
             <div class="panel panel-default">
                 <div class="panel-heading">Dashboard</div>
 
                 <div class="panel-body">
                     You are logged in!
                 </div>
             </div>
+        </div> -->
```
Comment (rosoarv) [inline resources/views/home.blade.php]: @litocabornay please remove commented code. thanks!

**Review by rosoarv — CHANGES_REQUESTED** [review]

**Review by rosoarv — APPROVED** [review]

**Comment by rosoarv** [comment]
@litocabornay nice. please merge to master. thanks!

**Pull request merged**